api/src/pcapi/core/users/api.py

```diff
@@ -1481,11 +1481,23 @@ def notify_users_before_deletion_of_suspended_account() -> None:
         transactional_mails.send_email_before_deletion_of_suspended_account(account)
 
 
+def has_processed_extract(user: users_models.User) -> bool:
+    for extract in user.gdprUserDataExtract:
+        if not extract.is_expired and not extract.dateProcessed:
+            return False
+    return True
```

Comment:
```suggestion
def has_unprocessed_extract(user: users_models.User) -> bool:
    for extract in user.gdprUserDataExtract:
        if not extract.is_expired and not extract.dateProcessed:
            return True
    return False
```
La fonction ne retourne pas vrai si il y a un extract processé mais Faux si il y a un extract unprocessed. Il vaudrait mieux changer le nom (attention j'ai aussi inversé son comportement)